src/pokemon_mood.c

```diff
@@ -4,14 +4,14 @@
 #include "constants/items.h"
 
 static const s8 sMoodModifiers[] = {
```

Comment:
Pro tip: If you select the diff lines related to the comment instead of the line immediately before, it would make it a lot easier to see what you want reverted from the conversation page.